### PR TITLE
Containerize mockoon-cli with README examples

### PR DESCRIPTION
**Pull request by andy-ta**
Containerize mockoon-cli.

A bit tricky since we have to provide our own export.json to it, but I provided a README that details how to build one's own Docker image with one's own export.json around a base mockoon-cli Docker image that I pre-built.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM node:12
```
Comment (mauriceackel) [inline Dockerfile]:
```suggestion
FROM node:12-alpine
```
I think it is save to use the smaller alpine bistro here, what do you think?

File: docker/example.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM andyta/mockoon-cli:latest
```
Comment (mauriceackel) [inline docker/example.Dockerfile:2]:
```suggestion
# If you've built the base container yourself, change this to the tag of it 
FROM andyta/mockoon-cli:latest
```

**Review by mauriceackel — CHANGES_REQUESTED** [review]
Hi @andy-ta 👋 Thank you for contributing 🎉. I made just two small suggestions, other than that, it looks good to me. 🚀

File: Dockerfile (hunk 1 above)
Comment (andy-ta) [reply Dockerfile]: I had tried that initially - didn't work. Tried it again, turns out it was just missing some dependencies to run the binary, which I added to this Dockerfile :) Thanks for the review

**Review by mauriceackel — APPROVED** [review]
LGTM, merging now 🚀

**Pull request merged**